diff --git a/package/kernel/linux/modules/other.mk b/package/kernel/linux/modules/other.mk
index 8947ba4..20d1d25 100644
--- a/package/kernel/linux/modules/other.mk
+++ b/package/kernel/linux/modules/other.mk
@@ -767,30 +767,6 @@ endef
 $(eval $(call KernelPackage,ikconfig))
 
 
-define KernelPackage/zram
-  SUBMENU:=$(OTHER_MENU)
-  TITLE:=ZRAM
-  DEPENDS:=+kmod-lib-lzo +kmod-lib-lz4
-  KCONFIG:= \
-	CONFIG_ZSMALLOC \
-	CONFIG_ZRAM \
-	CONFIG_ZRAM_DEBUG=n \
-	CONFIG_PGTABLE_MAPPING=n \
-	CONFIG_ZSMALLOC_STAT=n \
-	CONFIG_ZRAM_LZ4_COMPRESS=y
-  FILES:= \
-	$(LINUX_DIR)/mm/zsmalloc.ko \
-	$(LINUX_DIR)/drivers/block/zram/zram.ko
-  AUTOLOAD:=$(call AutoLoad,20,zsmalloc zram)
-endef
-
-define KernelPackage/zram/description
- Compressed RAM block device support
-endef
-
-$(eval $(call KernelPackage,zram))
-
-
 define KernelPackage/mvsdio
   SUBMENU:=$(OTHER_MENU)
   TITLE:=Marvell SDIO support
